symbl/streaming_api/StreamingConnection.py

```diff
@@ -63,7 +64,11 @@ def send_audio(self, data):
 
     @wrap_keyboard_interrupt
     def send_audio_from_mic(self, device=None):
-        import sounddevice as sd
-        with sd.InputStream(blocksize=4096, samplerate=44100, channels=1, callback= lambda indata, *args: self.send_audio(indata.copy().tobytes()), dtype='int16', device=device):
-            while True:
-                pass
\ No newline at end of file
+        import pyaudio
+        audio = pyaudio.PyAudio()
+        chunk = 4096
+        samplerate = self.start_request['config']['speechRecognition']['sampleRateHertz']
```

Comment:
can we have a default value here? Also a null check in place, with a validation? Otherwise it will make this a mandatory field and existing snippets won't work if they don't have this field.

Reply:
Updated.